artview/components/navigator.py

```diff
@@ -47,7 +465,7 @@ def __init__(self, Vradar=None, Vgrid=None, Vfilelist=None,
             Parent instance to associate to this class.
             If None, then Qt owns, otherwise associated w/ parent PyQt instance
         '''
-        super(FileNavigator, self).__init__(name=name, parent=parent)
+        super(FileNavigator_old, self).__init__(name=name, parent=parent)
         self.central_widget = QtWidgets.QWidget()
```

Comment:
Just trying to get a hold here, what is the difference between the `FileNavigator` and `FileNavigator_old`